tests/integrations/django/test_basic.py

```diff
@@ -186,49 +186,80 @@ def test_sql_queries(sentry_init, capture_events):
 @pytest.mark.django_db
 def test_sql_dict_query_params(sentry_init, capture_events):
     sentry_init(integrations=[DjangoIntegration()], send_default_pii=True)
-    from django.db import connection
 
-    sql = connection.cursor()
+    from django.db import connections
+
+    sql = connections["postgres"].cursor()
 
     events = capture_events()
-    with pytest.raises(OperationalError):
-        # This really only works with postgres. sqlite will crash with syntax error
+    with pytest.raises(ProgrammingError):
         sql.execute(
             """SELECT count(*) FROM people_person WHERE foo = %(my_foo)s""",
             {"my_foo": 10},
         )
 
     capture_message("HI")
-
     event, = events
 
     crumb, = event["breadcrumbs"]
-    assert crumb["message"] == ("SELECT count(*) FROM people_person WHERE foo = 10")
```

Comment:
I don't think we want this change. Why does this happen?

Reply:
Yeah, I'm not sure why it's getting formatted funny in the message with the bytestring like that.  Fair enough that it's not what's desired

Reply:
I'll try to debug this.